tests/api.c

```diff
@@ -14824,7 +14824,8 @@ static void test_wc_PKCS7_EncodeEncryptedData (void)
                 NULL, sizeof(decoded)), BAD_FUNC_ARG);
     AssertIntEQ(wc_PKCS7_DecodeEncryptedData(&pkcs7, encrypted, encryptedSz,
                 decoded, 0), BAD_FUNC_ARG);
-    /* Test struct fields */
+    /* Test
```

Comment:
Please restore this comment to its previous one-line state.

Reply:
thanks